projects/1.20-fabric/assets/create-food/createfood/lang/zh_cn.json

```
@@ -0,0 +1,1729 @@
+  "item.createfood.melon_juice_bucket": "西瓜汁桶",
+  "item.createfood.melon_milkshake_bottle": "西瓜奶昔",
+  "item.createfood.melon_milkshake_bucket": "西瓜奶昔桶",
+  "item.createfood.milk_powder": "奶粉",
+  "item.createfood.milkshake_bottle": "奶昔",
+  "item.createfood.milkshake_bucket": "奶昔桶",
+  "item.createfood.minced_dragon": "龙肉末",
+  "item.createfood.mini_chocolate_graham_cracker_pie_crust": "迷你巧克力全麦粉饼干馅饼酥皮",
+  "item.createfood.mini_chocolate_pie_graham_cracker": "迷你巧克力派（全麦粉饼干）",
+  "item.createfood.mini_cookie_cream_pie": "迷你曲奇奶油派",
+  "item.createfood.mini_cream_pie_chocolate_graham_cracker": "迷你奶油派（巧克力全麦粉饼干）",
+  "item.createfood.mini_cream_pie_graham_cracker": "迷你奶油派（全麦粉饼干）",
+  "item.createfood.mini_graham_cracker_pie_crust": "迷你全麦粉饼干馅饼酥皮",
+  "item.createfood.mini_smores_pie": "迷你棉花糖夹心饼干派",
+  "item.createfood.molasses_bottle": "糖蜜",
+  "item.createfood.molasses_bucket": "糖蜜桶",
+  "item.createfood.mushroom_bacon_pizza_slice": "蘑菇培根披萨切片",
+  "item.createfood.mushroom_fish_pizza_slice": "蘑菇鱼肉披萨切片",
+  "item.createfood.mushroom_onion_pizza_slice": "蘑菇洋葱披萨切片",
+  "item.createfood.mushroom_pizza_slice": "蘑菇披萨切片",
+  "item.createfood.mutton_sandwich": "羊肉三明治",
+  "item.createfood.mutton_sandwich_beetroot": "羊肉三明治（甜菜根）",
+  "item.createfood.nacho_bowl": "芝士玉米片碗",
+  "item.createfood.onion_bacon_pizza_slice": "洋葱培根披萨切片",
+  "item.createfood.onion_pizza_slice": "洋葱披萨切片",
+  "item.createfood.orange_gelatin_dessert_slice": "橙色明胶甜品切片",
+  "item.createfood.orange_gelatin_mix_bucket": "橙色明胶混合桶",
+  "item.createfood.paprika": "红辣椒粉",
+  "item.createfood.pasta": "意面",
+  "item.createfood.pasta_plate": "意面盘",
+  "item.createfood.pasta_plate_butter": "黄油意面",
+  "item.createfood.pasta_plate_eggplant": "茄子意面",
+  "item.createfood.pasta_plate_endermite_meatballs": "末影螨肉丸意面",
+  "item.createfood.pasta_plate_endermite_meatballs_tomato_sauce": "末影螨肉丸番茄酱意面",
+  "item.createfood.pasta_plate_fish": "鱼肉意面",
+  "item.createfood.pasta_plate_meatballs": "肉丸意面",
+  "item.createfood.pasta_plate_mutton_chop": "羊排意面",
+  "item.createfood.pasta_plate_slime": "黏液意面",
+  "item.createfood.pasta_plate_slimeballs": "黏液球意面",
+  "item.createfood.pasta_plate_squid_ink": "鱿鱼墨汁意面",
+  "item.createfood.pasta_plate_strider_meatballs": "炽足兽肉丸意面",
+  "item.createfood.pasta_plate_strider_meatballs_tomato_sauce": "炽足兽肉丸番茄酱意面",
+  "item.createfood.pasta_plate_tomato_sauce": "番茄酱意面",
+  "item.createfood.pastry_bar_base": "酥皮条坯",
+  "item.createfood.pastry_base": "油酥糕点坯",
+  "item.createfood.peanut_butter_apple_jam_sandwich": "花生酱苹果酱三明治",
+  "item.createfood.peanut_butter_chorus_fruit_jam_sandwich": "花生酱紫颂果酱三明治",
+  "item.createfood.peanut_butter_melon_jam_sandwich": "花生酱西瓜酱三明治",
+  "item.createfood.pink_gelatin_dessert_slice": "粉红色明胶甜品切片",
+  "item.createfood.pink_gelatin_mix_bucket": "粉红色明胶混合桶",
+  "item.createfood.potato_chip_bowl": "薯片碗",
+  "item.createfood.potato_chips": "薯片",
+  "item.createfood.powdered_sugar": "糖粉",
+  "item.createfood.pressed_cocoa": "扁平可可",
```

```suggestion
  "item.createfood.pressed_cocoa": "压制可可",
```
不太清楚可可相关的加工流程，只能瞎猜一个了